src/apps/applets/db/schemas/applet.py

```diff
@@ -36,6 +44,8 @@ class AppletSchema(_BaseAppletSchema, Base):
     link = Column(UUID(as_uuid=True), unique=True)
     require_login = Column(Boolean(), default=True)
     pinned_at = Column(DateTime(), nullable=True)
+    retention_period = Column(Integer(), nullable=True)
+    retention_type = Column(String(20), nullable=True)
```

Comment:
Don't you think we should move these fields out from the Applet schema?

Reply:
it is directly related to applet, @dparfeniuk why do you think so?

Reply:
It is just a question :)
I don't have a solid opinion on that. You can leave it as it is